soh/src/overlays/actors/ovl_Obj_Comb/z_obj_comb.c

```diff
@@ -154,6 +180,10 @@ void ObjComb_Init(Actor* thisx, PlayState* play) {
     Collider_InitJntSph(play, &this->collider);
     Collider_SetJntSph(play, &this->collider, &this->actor, &sJntSphInit, this->colliderItems);
     ObjComb_SetupWait(this);
+    if (gSaveContext.n64ddFlag) {
+        s16 respawnData = gSaveContext.respawn[RESPAWN_MODE_RETURN].data & ((1 << 8) - 1);
```

Comment:
Might want to leave a comment explaining the bitshift.

Reply:
I copied it from deku scrubs, which use the same bitshift, they also have this comment but it seems to be outdated:
```cpp
// Ugly, but the best way we can identify which grotto we are in, same method 3DRando uses, but we'll need to account for entrance rando
```